lib/osm_community_index/local_chapter.rb

```diff
@@ -0,0 +1,64 @@
+module OsmCommunityIndex
+  class LocalChapter
+    attr_reader :id, :url
+
+    def initialize(id, url)
+      @id = id
+      @url = url
+    end
+
+    def self.local_chapters
+      @chapters = init_local_chapters
```

Comment:
This has two problems - it's not actually memoising (should be `||=`) and it's using a class variable which is not thread safe. Although that doesn't matter here (perhaps), if we include it someone else will copy this approach elsewhere at some point.

Reply:
🙄 I removed the memoising so I could test changes I was making, and once again forgot to add it back before pushing 🙄